CHANGELOG.rst

```diff
@@ -2,16 +2,22 @@
 Change history
 ==============
 
-3.x.x (2025-03-31)
+3.0.2 (2025-03-31)
 ------------------
 
 **Project maintenance**
-
-* Confirm support for Postgres 17 and drop (verified) support for Postgres 12
+* upgraded docker image to debian-bookworm [open-api-framework/#125]
```

Comment:
Could you make links of these references by using `extlinks`? `objects-api` has this configured already:

https://github.com/maykinmedia/objects-api/blob/0d42890323ee1cd14760a1af9fb5af149cbf47e7/docs/conf.py#L107
https://github.com/maykinmedia/objects-api/blob/0d42890323ee1cd14760a1af9fb5af149cbf47e7/docs/conf.py#L49

You can then reference like this: https://github.com/maykinmedia/objects-api/blob/0d42890323ee1cd14760a1af9fb5af149cbf47e7/CHANGELOG.rst?plain=1#L22

Reply:
object-types does not have documentation, everything is located in the objects-api repo.